test/test_utils/ec2.py

```diff
@@ -509,6 +513,46 @@ def enforce_IMDSv2(instance_id, region=DEFAULT_REGION, ec2_client=None, hop_limi
     LOGGER.info(f"Modify Metadata options State of EC2 instance: {state}")
 
 
+
+def enforce_IMDSv1(instance_id, region=DEFAULT_REGION, ec2_client=None):
+    """
+    Enabled HTTP TOKENS required option on EC2 instance with given hop limit.
```

Comment:
will change this text to below after initial PR review for source code

`Enable IMDSv1 on EC2 instance`